api.js

```diff
@@ -1,17 +1,27 @@
-import reducer from './lib/reducers';
-import updateStore from './lib/sync/update-store';
-import updateStorageManager from './lib/middleware/update-storage-manager';
-import makeBrainstemType from './lib/types/make-brainstem-type';
-import modelActions from './lib/actions/model';
-import collectionActions from './lib/actions/collection';
-import stopUpdateStore from './lib/sync/stop-update-store';
+import reReducer from './lib/reducers';
+import usUpdateStore from './lib/sync/update-store';
```

Comment:
Can you tell me more about the prefixes in this PR? We usually stay away from Hungarian notation (or any kind of notation) so I would like to know why we shifted away from convention in this PR.

Reply:
Its been a bit but I think it was something with duplicate names? I am not attached to it in any way just trying to get it working.

Reply:
Yes, duplicate names. So I was just randomly using a convention to pull in `foo` as `iFoo` or whatever...sorry it looks like hungarian notation (I hate that too!).

It turns out to be quite fussy importing and then exporting. In fact, just doing a pure "re-export" ends up screwing up things often.